bin/contentctl_project/contentctl_core/domain/entities/detection.py

```diff
@@ -112,6 +112,14 @@ def search_validation(cls, values):
                     raise ValueError('Use source macro instead of eventtype, sourcetype, source or index in detection: ' + values["name"])
         return values
 
+    @root_validator
+    def name_max_length(cls, values):
```

Comment:
@P4T12ICK : Hey, i added the name_max_length function under root validator and only checks for ESCU search name length. I tested this and it works correctly, do you think this is how you'd do this check?